src/palette/qml/MuseScore/Palette/internal/PaletteTree.qml

```diff
@@ -116,6 +116,7 @@ StyledListView {
             // in search and no cell selected: apply the first found element
             const parentIndex = paletteTreeDelegateModel.modelIndex(0);
             index = paletteModel.index(0, 0, parentIndex);
+            paletteSelectionModel.select(index, ItemSelectionModel.Select);
```

Comment:
This will give visual feedback (unlike in MS3): highlight the first item when search's onCompleted during application of the element onto score.